### PR TITLE
Fixed math and minor errors in docstrings for Read the Docs

### LINKED ISSUE #155: Fix math in docstrings so that it shows up correctly on Read the Docs
Several mathematical expressions were included in docstrings before we had completely set up the [PlasmaPy Read the Docs page](http://plasmapy.readthedocs.io/en/latest/).  Some of these untested docstrings are rendering incorrectly on the documentation page.  We will need to go through and change the docstrings so that they do show up correctly.  The following pages look like they require fixes:

- [Math](http://plasmapy.readthedocs.io/en/latest/math/index.html) from [`math.py`](https://github.com/PlasmaPy/PlasmaPy/blob/master/plasmapy/math/math.py)
- [Physics](http://plasmapy.readthedocs.io/en/latest/physics/index.html) from [`parameters.py`](https://github.com/PlasmaPy/PlasmaPy/blob/master/plasmapy/physics/parameters.py), [`quantum.py`](https://github.com/PlasmaPy/PlasmaPy/blob/master/plasmapy/physics/quantum.py), [`relativity.py`](https://github.com/PlasmaPy/PlasmaPy/blob/master/plasmapy/physics/relativity.py), and [`transport.py`](https://github.com/PlasmaPy/PlasmaPy/blob/master/plasmapy/physics/transport.py)

Thank you!

### PR DESCRIPTION
**Pull request by jasperbeckers**
Closes #155 
Changed some of the math formatting in the docstrings so that they appear correctly on Read the Docs, and fixed a few small other formatting errors too. A test version for the updated docs is [here](http://plasmapy-fixed-math-test.readthedocs.io/en/fixing_rtd_math/)!

### CODE REVIEW

**Comment by StanczakDominik** [comment]
Thanks for doing this one! I'll look at it later today and likely merge the
changes.

It's kinda weird to have to set up custom RTD pages for pull requests, but
I just checked back with them and it seems this is the state of the world
right now. For now it's either this or local Sphinx. See the docs for that
one, if they're not clear enough on how to build it I'd appreciate pointing
any flaws out.

File: plasmapy/physics/distribution.py [hunk 1]
```diff
@@ -37,8 +37,7 @@ def Maxwellian_1D(v: units.m/units.s,
     Returns
     -------
     f : Quantity
-        probability in Velocity^-1, normized so that:
-        $\int_{- \infty}^{\infty} f(v) dv = 1}
+        probability in Velocity^-1, normized so that: :math:`\int_{-\infty}^{+\infty} f(v) dv = 1`
```
Comment (StanczakDominik) [inline plasmapy/physics/distribution.py]: Some minor errors in the original version of this line but #152 is working on that so don't worry about it :)

File: plasmapy/physics/distribution.py [hunk 2]
```diff
@@ -58,12 +57,11 @@ def Maxwellian_1D(v: units.m/units.s,
     In one dimension, the Maxwellian distribution function for a particle of
     mass m, velocity v, a drift velocity V and with temperature T is:
 
-    .. math::
-    f = \sqrt{\frac{m}{2 \pi k_B T}} \exp(-\frac{m}{2 k_B T} (v-V)^2)
+    :math:`f = \sqrt{\frac{m}{2 \pi k_B T}} e^{-\frac{m}{2 k_B T} (v-V)^2}`
```
Comment (StanczakDominik) [inline plasmapy/physics/distribution.py]: I'm noticing that `:math:` seems not to center the formulas. I'm guessing `.. math ::` would do that, thus the two would be somewhat equivalent to LaTeX's `$` and `$$` envelopes. Could you try this with `.. math ::` one more time, maybe google would help? I seem to recall seeing this `.. math ::` other libraries' documentation, so I think we might have been doing something else wrong.

**Comment by jasperbeckers** [comment]
Yes from what I can see .. math:: is simply the latex equivalent to \begin{equation} with ::math:: being the inline variant like $\pi$.

Indentation could certainly explain it not working, I'll play with that tomorrow. Nice catch on them not being centered anymore. It makes sense if you think about it, haha

File: plasmapy/math/math.py [hunk 3]
```diff
@@ -115,8 +113,7 @@ def plasma_dispersion_func_deriv(zeta):
     -----
     The plasma dispersion function is defined as:
```
[the remark this answers is not in the document]
Comment (jasperbeckers) [reply plasmapy/math/math.py]: Are we sure the equation written in the docstring is correct? It uses a factor of pi^0.5, whereas online I find factors of pi^-0.5.

[the remark this answers is not in the document]
Comment (StanczakDominik) [reply plasmapy/math/math.py]: I went back to the original source in the function (Fried and Conte), and it turns out that yeah, that was a bug. The formula itself will also need fixing, not just the docstring!

**Comment by jasperbeckers** [comment]
@StanczakDominik Thanks for the approval. Please go ahead, I am not familiar with the theory at all!

**Comment by StanczakDominik** [comment]
I'm not either! In fact, I got confused myself for a second there. It was right all along!
Here's our code:

    Z = 1j * np.sqrt(np.pi) * np.exp(-zeta**2) * (1.0 + special.erf(1j * zeta))

And here's a little bit of Fried and Conte:

image

I suppose you might have been looking for the formula with the integral in it, [as seen here](https://farside.ph.utexas.edu/teaching/plasma/lectures1/node87.html). I know I got confused by that!

**Comment by jasperbeckers** [comment]
Oh, the actual equation used in the code is indeed correct! Sorry if I was not clear about that. It was only the docstring integrals that were incorrect.

**Comment by StanczakDominik** [comment]
Oh crud, we do have the version with the integral in the docstring! My bad, sorry!

**Comment by jasperbeckers** [comment]
Can we close this or does it require more work?

**Comment by StanczakDominik** [comment]
Oh, sorry, I probably hit that approve button a bit too soon. Would you be
okay with going through the math tags again and converting them to blocks
like we figured out works? If it's too much to ask, as everyone hates
busywork so I'd understand, I'll do that tomorrow.

Unless that's already in, hard to tell from my phone!

**Comment by jasperbeckers** [comment]
Oh absolutely, I was just unsure whether we were going with display or inline mode :) That answers the question!

**Pull request merged**